pkg/controller/logstash/logstash_controller_test.go

```diff
@@ -201,6 +210,14 @@ func TestReconcileLogstash_Reconcile(t *testing.T) {
 				Spec: logstashv1alpha1.LogstashSpec{
 					Version: "8.6.1",
 					Count:   1,
+					UpdateStrategy: appsv1.StatefulSetUpdateStrategy{
+						Type: appsv1.RollingUpdateStatefulSetStrategyType,
+						RollingUpdate: &appsv1.RollingUpdateStatefulSetStrategy{
+							MaxUnavailable: &intstr.IntOrString{
+								IntVal: 1,
+							},
+						},
+					},
```

Comment:
Not from this PR but I am a bit confused by this test. Not sure what it actually verifies. It seems it it is just retrieving the Logstash resource again that we just put in as an expected object.

Reply:
It verifies the value of LogstashStatus is up-to-date

Reply:
I see. But for the addition of the `UpdateStrategy` a test would be more useful that verifies that the `UpdateStrategy` specified in the Logstash resource is actually applied to the `StatefulSet` that is being created.

Reply:
test is updated